app/models/user.rb

```diff
@@ -1,16 +1,22 @@
 # frozen_string_literal: true
 require 'common/models/base'
+require 'mvi/messages/find_candidate_message'
+require 'mvi/service'
 
 class User < RedisStore
   NAMESPACE = REDIS_CONFIG['user_store']['namespace']
   REDIS_STORE = Redis::Namespace.new(NAMESPACE, redis: Redis.current)
   DEFAULT_TTL = REDIS_CONFIG['user_store']['each_ttl']
+  MVI_SERVICE = VetsAPI::Application.config.mvi_service
 
   # id.me attributes
   attribute :uuid
   attribute :email
   attribute :first_name
+  attribute :middle_name
   attribute :last_name
+  attribute :gender
+  attribute :dob, Common::UTCTime
```

Comment:
Then again, this work overlaps with what I'm doing.  Is there any motivation from the MVI world to converting this value to something other than just a simple string?